Gemfile

```diff
@@ -32,6 +32,12 @@ gem "puma", "~> 2.16.0"
 
 gem 'pg', platforms: :ruby
 
+group :production, :staging do
```

Comment:
we shouldn't need these. They are for connecting to VACOLS. Feedback will not be doing that.